### ISSUE TITLE
Album Content layout

### ISSUE THREAD

**Issue by alyons-dm** [Digitalminion/DM-ng-legacy-foolish-endeavor#40]
<img width="413" alt="screen shot 2017-04-18 at 9 11 51 pm" src="https://cloud.githubusercontent.com/assets/20231883/25163152/06c14b80-247d-11e7-9fd6-60fbdbd6294d.png">

Do I make a new page to contain photos? one for albums?

**Comment by jfn6030217** [comment]
In a situation like this, it would be appropriate to have a new template for "album detail view", which is basically the same as the template for "album overview".  The two templates could be basically the same, except for how the flexbox lays out the items in it.  This seems like it would be less of a hassle.  Austin may have good ideas on this, however.

**Comment by jfn6030217** [comment]
We could safely say that this ticket is blocked by #38 -- that is to say, you should complete #38 before you begin work on this one, to make it easier on yourself.

**Comment by alyons-dm** [comment]
I am making more comprehensively broken down tickets so I will be closing this for the sake of a clean start. a similar concept may be revisited in a new ticket.
